src/main/java/bowling/model/frame/NormalFrame.java

```diff
@@ -9,6 +9,8 @@
 	private static final String FRAME_RANGE_ERROR_MESSAGE = "일반 프레임은 1~9 까지 입니다.";
 	private static final int MAX_FRAME_NUMBER = 9;
 	private static final int MIN_FRAME_NUMBER = 1;
+	private static final int CUSTOM_SECTION_POINT = -1;
+	private static final int FIRST_INDEX = 0;
 
 	private final NormalPlay normalPlay;
```

Comment:
```suggestion
	private final playable normalPlay;
```

> playable은 다형성을 위해서 공통된 규약때문에 사용했습니다.

다형성과 공통된 규약을 위해 사용하셨다면, 해당 다형성과 규약의 강제의 의점을 얻을 수 있도록 설계해보면 어떨까요!?

Reply:
인터페이스를 사용한건 추상 메서드의 특정 기능을 하위 클래스로 반드시 강제로 구현을 해야되서 사용한건데
제가 만들어낸 위의 타입은 자식 타입만 선언을 해놔서 제가 시간이 지나 다시 봤을땐 normalPlay가 상위의 playable을
알수있는지는 모르는 구조네요